report/src/bib/mybib.bib

```diff
@@ -398,3 +398,22 @@ @article{longoAccurateOccupancyEstimation2019
   urldate      = {2023-03-30},
   langid       = {english}
 }
+
+@article{gruberCO2SensorsOccupancy2014,
+  title        = {{{CO2}} Sensors for Occupancy Estimations: {{Potential}} in Building Automation Applications},
+  shorttitle   = {{{CO2}} Sensors for Occupancy Estimations},
+  author       = {Gruber, Mattias and Trüschel, Anders and Dalenbäck, Jan-Olof},
+  date         = {2014-12-01},
+  journaltitle = {Energy and Buildings},
+  shortjournal = {Energy and Buildings},
+  volume       = {84},
+  pages        = {548--556},
+  issn         = {0378-7788},
+  doi          = {10.1016/j.enbuild.2014.09.002},
+  url          = {https://www.sciencedirect.com/science/article/pii/S0378778814006914},
+  urldate      = {2023-04-06},
+  abstract     = {It is well established that model-based controllers with an integrated control model and information about indoor climate disturbances have the ability to drastically improve the performance of building automation systems. In offices, occupancy is one of the most important disturbances to account for in this context, but the available methods for accurately determining the number of people are commonly too complex for considering at most sites. However, since model-based controllers can be made robust, input deficiencies can to a certain degree be compensated for, and in this work, a two step procedure was applied to investigate the potential of utilizing simplified estimations generated from CO2 sensor responses. First, the expected time delay and error of such estimations were derived experimentally for various occupancy changes and ventilation flow rates. Next, an office site was simulated and the occupancy information used by a model-based controller for ventilation control was stepwise subjected to various errors and time delays by considering the expected values as references. The results showed that the estimations in many case were sufficient for achieving a high control performance, but beyond a certain level, the deficiencies could only be met by an increased complexity of the controller.},
+  langid       = {english},
+  keywords     = {📖,📚,Building automation system,Controller robustness,Heating; ventilation and air-conditioning,Indoor air quality,Information deficiencies,Model predictive control,Occupancy estimations,Office buildings,Predictive control},
```

Comment:
why emoji

Reply:
Good catch - it's just my Zotero filing system.